tests/unit/builders/indexable-post-builder-test.php

```diff
@@ -18,8 +20,8 @@
 /**
  * Class Indexable_Post_Builder_Test.
  *
- * @group indexables
- * @group builders
+ * @group  indexables
```

Comment:
Is there a specific reason for adding these spaces?

Reply:
The auto-formatting feature in my PHPStorm seems to think so...
I will check if I can teach my PHPStorm not to do this.